apps/src/templates/Lightbulb.jsx

```diff
@@ -1,15 +1,18 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import Radium from 'radium'; // eslint-disable-line no-restricted-imports
```

Comment:
I followed the convention used in `ChatBubbleTip`. Not sure if I need to implement a different way?